css-fonts/styles.css

```diff
@@ -0,0 +1,13 @@
+body {
+  line-height: 1.6em;
+}
+
+body p {
```

Comment:
In this instance, you don't actually need to target the `p` here, you can just select the `body`.